packages/@okta/vuepress-site/guides/build-self-signed-jwt/overview/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Overview
+---
+JSON Web Tokens (JWTs) allow claims, such as user data, to be represented in a secure manner, helping to ensure trust and security in your app. JWTs are an open standard and there are various libraries available that allow you to create, verify, and inspect them.
+
```

Comment:
We need a little more explanation of the "big picture" here ("use this guide when..."). Feel free to wordsmith this:


---

Okta uses JWTs for many purposes. One purpose is to add an optional additional layer of security to requests your code makes to Okta's [OpenID Connect](https://developer.okta.com/docs/api/resources/oidc/) endpoints that require [client authentication](https://developer.okta.com/docs/api/resources/oidc/#client-authentication-methods), like [/token](https://developer.okta.com/docs/api/resources/oidc/#token).

Follow this guide to understand:
* How to build a signed JWT to use when making requests to these endpoints
* Which claims are required in your JWT payload
* How to sign the JWT with a shared secret or private key